packages/plugin-conversation/src/process-image.browser.js

```diff
@@ -6,9 +6,7 @@
 
 import {pick} from 'lodash';
 import {orient} from '@ciscospark/helper-image';
-import {base64} from '@ciscospark/common';
-
-/* global Blob, document, Image, URL */
+/* eslint-env browser */
```

Comment:
Nice. I feel like this didn't work last time. Or, didn't have *all* the globals.